src/pages/index.js

```diff
@@ -107,18 +134,86 @@ profileAddButton.addEventListener("click", function () {
 
 /* Functions */
 
+function handleAvatarUpdate({ link }) {
+  return api
+    .updateAvatar({ avatar: link })
+    .then((res) => {
+      userInfo.setUserInfo({
+        title: res.name,
+        description: res.about,
+        avatar: res.avatar,
+      });
+      setAvatarPopup.close();
+    })
+    .catch((err) => console.err(`Error updating avatar: ${err}`));
+}
+
+function cardLikeClick(cardId, isLiked) {
+  if (isLiked) {
+    api.likeCard(cardId);
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**